homeassistant/components/kat_bulgaria/binary_sensor.py

```diff
@@ -0,0 +1,63 @@
+"""Binary sensor platform."""
+
+from datetime import timedelta
+import logging
+
+from kat_bulgaria.obligations import KatApi, KatApiResponse
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .common import generate_entity_name
+from .const import CONF_DRIVING_LICENSE, CONF_PERSON_EGN, CONF_PERSON_NAME, DOMAIN
+
+SCAN_INTERVAL = timedelta(minutes=20)
```

Comment:
```suggestion
SCAN_INTERVAL = timedelta(days=1)
```

Reply:
The idea is to get your notifications faster and know you have been fined sooner. 

Plus the API goes down multiple times a day. My python package filters all exceptions and presents the response to HA. If there is an error = skip the update.

I thought about implementing retries, however the API goes down for 2-30 minutes at a time. This means if the API throws an error 2 times in a row, you will get your update after 3x20 = 60 min with the current settings and after 3 days with your suggestion.

That's why a lower interval is better in this case. I can increase it to 30 minutes or an hour but I prefer more frequent checks.

Reply:
P.S. I know all of this, because I created a microservice that makes requests every 10 minutes for 2 different people.
It logs the response and the request duration. I can show the logs if needed.

The service has been running since January, so I have a LOT of test data against that API.
It also doesn't seem to have any throttling either, I tested with 15-second delay for a couple of days and everything was fine.

Reply:
I've discussed this PR with the other members in the home-assistant/core team, and considering the API is that unstable, we don't think this integration is a good fit for core. We recommend you to use it as a custom integration instead.